DDC generation fails for 4.27.0 on Windows due to opencv_world311.dll missing dependencies

```
ue4-docker version:         0.0.89 (latest available version is 0.0.89)
Operating system:           Windows Server 2019 Standard (Build 17763.1935)
Docker daemon version:      20.10.5
NVIDIA Docker supported:    No
Maximum image size:         400GB
Available disk space:       644.51 GiB
Total system memory:        16 GiB physical, 36 GiB virtual
Number of processors:       9 physical, 18 logical
```

Additional details:

- Are you accessing the network through a proxy server? **No**

Command Used:
```ue4-docker build 4.27.0 --visual-studio=2019 --no-engine --exclude debug --exclude templates --monitor```

Initial Build Log:
```
[ue4-docker build] COMMAND-LINE INVOCATION:
[ue4-docker build] ['ue4-docker', '4.27.0', '--visual-studio=2019', '--no-engine', '--exclude', 'debug', '--exclude', 'templates', '--monitor', '-username', 'CanisHelix', '-password', '*******', '--verbose']

[ue4-docker build] UNREAL ENGINE VERSION SETTINGS:
[ue4-docker build] Custom build:  No
[ue4-docker build] Release:       4.27.0
[ue4-docker build] Repository:    https://github.com/EpicGames/UnrealEngine.git
[ue4-docker build] Branch/tag:    4.27.0-release

[ue4-docker build] ADVANCED CONFIGURATION OPTIONS:
[ue4-docker build] buildgraph_args: " -set:VS2019=true"
[ue4-docker build] excluded_components: {"ddc": false, "debug": true, "templates": true}

[ue4-docker build] WINDOWS CONTAINER SETTINGS
[ue4-docker build] Isolation mode:               process
[ue4-docker build] Base OS image:                mcr.microsoft.com/windows/servercore:ltsc2019
[ue4-docker build] Dll source image:             mcr.microsoft.com/windows:1809
[ue4-docker build] Host OS:                      Windows Server 2019 Standard (Build 17763.1935)
[ue4-docker build] Memory limit:                 No limit
[ue4-docker build] Detected max image size:      400GB
[ue4-docker build] Visual Studio:                2019
[ue4-docker build] GENERAL SETTINGS
[ue4-docker build] Excluding the following Engine components:
[ue4-docker build] - Debug symbols
[ue4-docker build] - Template projects and samples
```

It fails after an overnight build with the final log of
```
  Generating DDC data for TP_InCamVFXBP on Windows+WindowsNoEditor
  Running UE4Editor DerivedDataCache for project C:\UnrealEngine\Templates\TP_InCamVFXBP\TP_InCamVFXBP.uproject
  Commandlet log file is C:\UnrealEngine\Engine\Programs\AutomationTool\Saved\DerivedDataCache-2021.09.02-07.15.25.txt
  Running: C:\UnrealEngine\LocalBuilds\InstalledDDC\Engine\Binaries\Win64\UE4Editor-Cmd.exe C:\UnrealEngine\Templates\TP_InCamVFXBP\TP_InCamVFXBP.uproject -run=DerivedDataCache  -TargetPlatform=Windows+WindowsNoEditor -fill -DDC=CreateInstalledEnginePak -ProjectOnly -abslog=C:\UnrealEngine\Engine\Programs\AutomationTool\Saved\DerivedDataCache-2021.09.02-07.15.25.txt -stdout -CrashForUAT -unattended -NoLogTimes
    LogConsoleResponse: Display: Failed to find resolution value strings in scalability ini. Falling back to default.
    LogConsoleResponse: Display: Failed to find resolution value strings in scalability ini. Falling back to default.
    LogInit: Display: Running engine for game: TP_InCamVFXBP
    LogInit: Display: Loading text-based GConfig....
    LogWindows: Error: begin: stack for UAT
    LogWindows: Error: === Critical error: ===
    LogWindows: Error:
    LogWindows: Error: Fatal error!
    LogWindows: Error:
    LogWindows: Error: Unhandled Exception: 0xc06d007e
    LogWindows: Error:
    LogWindows: Error: [Callstack] 0x00007ffa563b9689 KERNELBASE.dll!UnknownFunction []
    LogWindows: Error: [Callstack] 0x00007ffa331b515a UE4Editor-OpenCVHelper.dll!__delayLoadHelper2() [d:\a01\_work\2\s\src\vctools\delayimp\delayhlp.cpp:312]
    LogWindows: Error: [Callstack] 0x00007ffa331b461d UE4Editor-OpenCVHelper.dll!_tailMerge_opencv_world331_dll() []
    LogWindows: Error: [Callstack] 0x00007ffa331b4096 UE4Editor-OpenCVHelper.dll!FOpenCVHelperModule::StartupModule() [C:\UnrealEngine\Engine\Plugins\Compositing\OpenCVLensDistortion\Source\OpenCVHelper\Private\OpenCVHelperModule.cpp:91]
    LogWindows: Error: [Callstack] 0x00007ffa46336c22 UE4Editor-Core.dll!FModuleManager::LoadModuleWithFailureReason() [C:\UnrealEngine\Engine\Source\Runtime\Core\Private\Modules\ModuleManager.cpp:538]
    LogWindows: Error: [Callstack] 0x00007ffa4c6261b3 UE4Editor-Projects.dll!FModuleDescriptor::LoadModulesForPhase() [C:\UnrealEngine\Engine\Source\Runtime\Projects\Private\ModuleDescriptor.cpp:643]
    LogWindows: Error: [Callstack] 0x00007ffa4c63c5ea UE4Editor-Projects.dll!FPluginManager::TryLoadModulesForPlugin() [C:\UnrealEngine\Engine\Source\Runtime\Projects\Private\PluginManager.cpp:1450]
    LogWindows: Error: [Callstack] 0x00007ffa4c625f30 UE4Editor-Projects.dll!FPluginManager::LoadModulesForEnabledPlugins() [C:\UnrealEngine\Engine\Source\Runtime\Projects\Private\PluginManager.cpp:1525]
    LogWindows: Error: [Callstack] 0x00007ff68b217baf UE4Editor-Cmd.exe!FEngineLoop::AppInit() [C:\UnrealEngine\Engine\Source\Runtime\Launch\Private\LaunchEngineLoop.cpp:5655]
    LogWindows: Error: [Callstack] 0x00007ff68b22e2d4 UE4Editor-Cmd.exe!FEngineLoop::PreInitPreStartupScreen() [C:\UnrealEngine\Engine\Source\Runtime\Launch\Private\LaunchEngineLoop.cpp:2224]
    LogWindows: Error: [Callstack] 0x00007ff68b220d37 UE4Editor-Cmd.exe!GuardedMain() [C:\UnrealEngine\Engine\Source\Runtime\Launch\Private\Launch.cpp:132]
    LogWindows: Error: [Callstack] 0x00007ff68b2210ea UE4Editor-Cmd.exe!GuardedMainWrapper() [C:\UnrealEngine\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:137]
    LogWindows: Error: [Callstack] 0x00007ff68b22410d UE4Editor-Cmd.exe!LaunchWindowsStartup() [C:\UnrealEngine\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:273]
    LogWindows: Error: [Callstack] 0x00007ff68b2354b4 UE4Editor-Cmd.exe!WinMain() [C:\UnrealEngine\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:320]
    LogWindows: Error: [Callstack] 0x00007ff68b2373e2 UE4Editor-Cmd.exe!__scrt_common_main_seh() [d:\a01\_work\2\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
    LogWindows: Error: [Callstack] 0x00007ffa624c7974 KERNEL32.DLL!UnknownFunction []
    LogWindows: Error: [Callstack] 0x00007ffa7051a2f1 ntdll.dll!UnknownFunction []
    LogWindows: Error:
    LogWindows: Error: end: stack for UAT
[ue4-docker build] [2021-09-02 07:15:29] [Available disk: 644.56 GiB] [Available memory: 12.61 GiB physical, 32.52 GiB virtual] [CPU usage: 22.90%]
  Took 4.182265s to run UE4Editor-Cmd.exe, ExitCode=3
  Editor terminated with exit code 3 while running DerivedDataCache for C:\UnrealEngine\Templates\TP_InCamVFXBP\TP_InCamVFXBP.uproject; see log C:\UnrealEngine\Engine\Programs\AutomationTool\Saved\Logs\BuildDerivedDataCache\DerivedDataCache-2021.09.02-07.15.29.txt
  AutomationTool exiting with ExitCode=1 (Error_Unknown)
Took 3475.0161021s to run AutomationTool.EXE, ExitCode=1
AutomationTool exiting with ExitCode=1 (Error_Unknown)
BUILD FAILED
```

I am re-building now after cleaning all images again with a pipe to build.log so that I can capture a full error log with the verbose option.

I have manually implemented https://github.com/adamrehn/ue4-docker/pull/195 in order to progress this far due previous failure, all images were cleaned after implementing the PR to ensure clean images.

--- a/ue4docker/dockerfiles/ue4-build-prerequisites/windows/Dockerfile
+++ b/ue4docker/dockerfiles/ue4-build-prerequisites/windows/Dockerfile
@@ -10,6 +10,7 @@ SHELL ["cmd", "/S", "/C"]
 # Gather the system DLLs that we need from the full Windows base image
 COPY --from=dlls `
     C:\Windows\System32\avicap32.dll `
+    C:\Windows\System32\avifil32.dll `
     C:\Windows\System32\avrt.dll `
     C:\Windows\System32\d3d10warp.dll `
     C:\Windows\System32\D3DSCache.dll `
@@ -20,6 +21,7 @@ COPY --from=dlls `
     C:\Windows\System32\mfplat.dll `
     C:\Windows\System32\mfplay.dll `
     C:\Windows\System32\mfreadwrite.dll `
+    C:\Windows\System32\msacm32.dll `
     C:\Windows\System32\msdmo.dll `
     C:\Windows\System32\msvfw32.dll `
     C:\Windows\System32\opengl32.dll `

